var/spack/repos/builtin/packages/intel-mpi/package.py

```diff
@@ -11,6 +11,8 @@ class IntelMpi(IntelPackage):
 
     homepage = "https://software.intel.com/en-us/intel-mpi-library"
 
+    version('2019.8.217', sha256='fa163b4b79bd1b7509980c3e7ad81b354fc281a92f9cf2469bf4d323899567c0',
```

Comment:
I think this should be `2020.2.254`.

Reply:
AFAIK there is no Intel MPI 2020.2.254. Parallel Studio 2020.2.254 is distributing Intel MPI 2019.8.
<img width="391" alt="IntelMPI2019 8" src="https://user-images.githubusercontent.com/15385762/88797103-162ace80-d1a3-11ea-8d80-fa113eb73c3f.png">

The next Intel MPI seems to be 2021 instead. Intel MPI versions are 2017, 2018, 2019 and 2020.
https://software.intel.com/content/www/us/en/develop/articles/intel-mpi-library-release-notes.html

Reply:
Hmm, I downloaded and installed this Intel MPI version and I swear it was 2020.2.254 in the path structure.

Reply:
I guess it should at least be `2019.8.254`?

Reply:
Yes, you are right. I have forgotten to update it to 254.

Reply:
Ok after looking further into this, the versions make sense to me now.